.docker/run-examples.sh

```diff
@@ -5,7 +5,7 @@
 # Input:
 #   $1 Log string.
 function log {
-  echo -e "[`TZ=Asia/Shanghai date`] ${1}"
+  echo -e "[`date`] ${1}"
```

Comment:
Is this type of codacy-bot feedback useful on style, or do you want to ignore it?

Codacy also has some backlogged complaints about not exporting script variables -- things like that are not important for functionality, but it could be done anyway for a cleaner code dashboard.

Reply:
I'd like to fix the issues in codacy in one PR. 🤔